README.md

```diff
@@ -10,7 +10,118 @@ both in CI environments and in command-line to produce a tile.
 
 ### `help`
 
+```
+$ kiln --help
+kiln
+kiln helps you build ops manager compatible tiles
+
+Usage: kiln [options] <command> [<args>]
+  --help, -h                                               bool    prints this usage information (default: false)
+  --pivotal-network-token, -pt, PIVOTAL_NETWORK_API_TOKEN  string  uaa access token for network.pivotal.io
+  --version, -v                                            bool    prints the kiln release version (default: false)
+
+Commands:
+  bake     bakes a tile
+  fetch    fetches releases
+  help     prints this usage information
+  update   updates stemcell_criteria and releases
+  version  prints the kiln release version
+```
+
+### `fetch`
+
+The `fetch` command downloads bosh release tarballs from an AWS S3 bucket to a
+a local directory specified by the `--releases-directory` flag. It discovers
+releases based on information from both the assets file and an assets lock file.
+The assets lock file name is expected to be a file in the same directory as the
+specified assets file with `lock` instead of `yaml` as the filename extension.
+The S3 object name is determined based on using regular expression capture
+groups.
+
+Kiln verifies that the checksum (SHA1) of the downloaded release matches
+checksum specified for the release in the assets lock file. If the checksums do
+not match, then the releases that don't match will be deleted from disk. *Since
+BOSH releases from different directors with the same packages result in complied
+releases with different hashes this may result in some problems for local
+development.*
+
+Kiln will not download releases if an existing release exists with the correct
+release version and checksum.
+
+#### Assets File
+The assets file must also have information about how to access the S3 Bucket.
+The following are **required** in the `compiled_releases` hash in an assets
+file.
+
+- `type`: must be "s3"
+- `bucket`: must be the name of the s3 bucket
+- `region`: must be the region of the bucket
+- `access_key_id`: must be an IAM access key id that has read permission for the
+  specified bucket
+- `secret_access_key`: must be the secret for the specified `access_key_id`
+- `regex:`: is used to specify a capture groups to map release and stemcell
```

Comment:
Suggestion: "a regular expression applied to the full-path directory listing of the S3 bucket.  Only files that match the regex will considered.  This regular expression must include four named capture groups..."

Reply:
I like how this reads better. Thanks!